src/GraphQL.Tests/Bugs/Issue1127.cs

```diff
@@ -49,11 +49,11 @@ public Issue127Query()
                 resolve: ctx =>
                 {
                     ctx.Arguments["s1"].ShouldBe("def1");
-                    ctx.Arguments["s2"].ShouldBe("def2");
+                    ctx.Arguments["s2"].ShouldBeNull();
```

Comment:
Since the query specified `s2: null`, it was testing for incorrect behavior.  Also, issue #1127 was opened because `ctx.Arguments` contained entries without the default values when an argument was not specified.  This is still true, so this change isn't really breaking issue #1127.